rplugin/python3/denite/source/menu.py

```diff
@@ -0,0 +1,66 @@
+# ============================================================================
+# FILE: menu.py
+# AUTHOR: Shougo Matsushita <Shougo.Matsu at gmail.com>
+#         TJ DeVries <devries.timothyj at gmail.com>
+# License: MIT license
+# ============================================================================
+
+from .base import Base
+
+
+class Source(Base):
+
+    def __init__(self, vim):
+        Base.__init__(self, vim)
+
+        self.name = 'menu'
+        self.kind = 'jump_list'
+
+        # self.matchers = []
+        # self.sorters = []
+
+    def on_init(self, context):
+        # TODO: Could also check for unite_source_menu_menus?
+        context['__menus'] = context.get('menus',
```

Comment:
To add `g:unite_source_menus` support is not bad.
But it supports Function type.
denite cannot support it.